app/components/UI/AssetOverview/AssetOverview.test.tsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import AssetOverview from './AssetOverview';
+import { zeroAddress } from 'ethereumjs-util';
+import renderWithProvider from '../../../util/test/renderWithProvider';
+import { backgroundState } from '../../../util/test/initial-root-state';
+import { NetworkController } from '@metamask/network-controller';
+import {
+  MOCK_ACCOUNTS_CONTROLLER_STATE,
+  MOCK_ADDRESS_2,
+} from '../../../util/test/accountsControllerTestUtils';
+import { fireEvent } from '@testing-library/react-native';
+
+const MOCK_CHAIN_ID = '0x1';
+
+const mockInitialState = {
+  engine: {
+    backgroundState: {
+      ...backgroundState,
+      PreferencesController: {
+        selectedAddress: MOCK_ADDRESS_2,
+      },
+      TokenRatesController: {
+        marketData: {
+          '0x1': {
+            [zeroAddress()]: { price: 0.005 },
+          },
+        },
+      },
+      NetworkController: {
+        providerConfig: {
+          chainId: MOCK_CHAIN_ID,
+        },
+      } as unknown as NetworkController['state'],
+      AccountsController: MOCK_ACCOUNTS_CONTROLLER_STATE,
+      AccountTrackerController: {
+        accountsByChainId: {
+          [MOCK_CHAIN_ID]: {
+            [MOCK_ADDRESS_2]: { balance: '0x1' },
+          },
+        },
+        // TODO: Replace "any" with type
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      } as any,
```

Comment:
```suggestion
      } as const,
```
I think this will work too

Reply:
done